<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SingleListWindow</class>
 <widget class="QDialog" name="SingleListWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>959</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Single Linked List Demo</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>941</width>
     <height>271</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="componentEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>261</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Organization</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>438</x>
     <y>20</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="projEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>261</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="methodEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>261</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Project</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Component</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Method</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Class</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="orgEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>261</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="classEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>261</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="randomButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>110</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Random</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="indexEdit">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>30</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelIndex">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>30</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Index</string>
   </property>
  </widget>
  <widget class="QPushButton" name="eraseButton">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>30</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Erase</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>10</y>
     <width>20</width>
     <height>161</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>80</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
